hud/src/androidLibMain/kotlin/AndroidHUD.kt

```diff
@@ -119,7 +166,9 @@ class AndroidHUD private constructor(viewResId: Int, hudConfig: HudConfig, priva
     }
 
     override fun setTitle(title: String?) {
-        loadingDialog.view?.findViewById<TextView>(R.id.text_view)?.text = title
-        loadingDialog.view?.findViewById<TextView>(R.id.text_view)?.visibility = if (title == null) View.GONE else View.VISIBLE
+        loadingDialog.view?.findViewById<TextView>(R.id.text_view)?.apply {
+            text = title
+            visibility = if (title == null) View.GONE else View.VISIBLE
+        }
```

Comment:
I guess this is still needed but it this will become legacy with a proper stack manager?

Reply:
This method is only to change title of the existing HUD

Reply:
Yes, but I believe it was added in lieu of proper stacking? the contract is kind of funky.

Reply:
Yes, it was added to avoid:
1. build with title1
1. show
1. hide
1. build with title2
1. show
1. hide

If you keep hud you can do
1. build with title1
1. show
1. set title to title2
1. hide

But how we can improve it?